tox.ini

```diff
@@ -17,6 +17,7 @@ passenv = CI TRAVIS TRAVIS_*
 changedir = {toxinidir}/output-{envname}
 deps =
     codecov
+    deprecated
```

Comment:
Since this is used in an actual plugin and not in a unit test, I think this dependency should be removed from here and placed in `requirements.txt`.

Reply:
If I remove this and try to run tox locally, it fails saying that it can't find deprecated. I'm not sure why as I have it installed.